DataFormats/simulation/src/MCUtils.cxx

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//
+// Created by Sandro Wenzel on 03.11.21.
+//
+
+#include <SimulationDataFormat/MCUtils.h>
+#include "TPDGCode.h"
+
+using namespace o2::mcutils;
+
+bool isStable(int pdg)
```

Comment:
We could expose this function in the header and reuse it in O2Physics "at least". What do you think?

Reply:
Yes, absolutely.